docs/APIs/security/tls.md

```diff
@@ -2,22 +2,22 @@
 
-<span class="notes">_**Note:** mbed TLS needs a secure source of random numbers, make sure that your target board has one and that it is fully ported to mbed OS. You can read more about this in our [porting guide](../../porting/tls_porting.md)._</span>
+<span class="notes">_**Note:** mbed TLS needs a secure source of random numbers; make sure that your target board has one and that it is fully ported to mbed OS. You can read more about this in our [porting guide](https://docs.mbed.com/docs/mbed-os-handbook/en/5.3/advanced/porting_guide/)._</span>
 
 ##Differences between the standalone and mbed OS editions
 
-mbed TLS has a standalone edition (available from the [mbed TLS website](https://tls.mbed.org/download)) for devices that are not running mbed OS. However, this guide focuses on the integration of mbed TLS into mbed OS. While the two editions share a code base, there are a number of differences, mainly in configuration and the platform integration. You should keep those differences in mind when reading articles in our [knowledge base](https://tls.mbed.org/kb), as currently all articles reference the standalone edition.
+mbed TLS has a standalone edition (available from the [mbed TLS website](https://tls.mbed.org/download)) for devices that are not running mbed OS. However, this guide focuses on the integration of mbed TLS into mbed OS. Although the two editions share a codebase, there are a number of differences, mainly in configuration and the platform integration. You should keep those differences in mind when reading articles in our [knowledge base](https://tls.mbed.org/kb), as currently all articles reference the standalone edition.
 
 The key differences are:
 
-- To reduce its footprint, the mbed OS edition enables a smaller set of features by default in the build configuration file `config.h`. While the default configuration of the standalone edition puts more emphasis on maintaining interoperability with a wide variety of platforms, the mbed OS edition only enables the most modern ciphers and most recent version of TLS and DTLS.
+- To reduce its footprint, the mbed OS edition enables a smaller set of features by default in the build configuration file `config.h`. Although the default configuration of the standalone edition puts more emphasis on maintaining interoperability with a wide variety of platforms, the mbed OS edition only enables the most modern ciphers and most recent version of TLS and DTLS.
 - The following components of mbed TLS are disabled in the mbed OS edition: `net.c` and `timing.c`. This is because equivalent platform code is available in mbed OS.
 
 ## Configuring mbed TLS features
 
-mbed TLS makes it easy to enable or disable any feature to meet the needs of a particular project, through compilation options. The default configuration:
+mbed TLS simplifies the process of enabling or disabling features to meet the needs of a particular project, through compilation options. The default configuration:
```

Comment:
Not sure we need the word "process", especially since at the end of the sentence we say that it's through complication options.